tests/functional/builtins/folding/test_slice.py

```diff
@@ -12,50 +12,45 @@
 @pytest.mark.fuzzing
 @settings(max_examples=50)
 @given(
-    a=st.integers(min_value=0, max_value=2**256 - 1),
-    s=st.integers(min_value=0, max_value=31),
-    le=st.integers(min_value=1, max_value=32),
+    bytes_in=st.binary(max_size=32),
+    start=st.integers(min_value=0, max_value=31),
+    length=st.integers(min_value=1, max_value=32),
 )
-def test_slice_bytes32(get_contract, a, s, le):
-    a = hex(a)
-    while len(a) < 66:
-        a = f"0x0{a[2:]}"
-    le = min(32, 32 - s, le)
+def test_slice_bytes32(get_contract, bytes_in, start, length):
+    as_hex = "0x" + str.join("", ["00" for _ in range(32 - len(bytes_in))]) + bytes_in.hex()
```

Comment:
you probably want ljust, rjust or zfill here

Reply:
Yes, `zfill` looks good, thanks!